tasks/debootstrap.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Debootstrap Ubuntu
+  command:
+    cmd: >
+      debootstrap --foreign --arch arm64
+      --variant minbase
+      --include systemd,ifupdown,openssh-server,python,dnsutils,udev
+      "{{ bootstrap_ubuntu_release | mandatory }}"
+      "{{ global_cache_dir | mandatory }}/rpi3-mount"
+       http://ports.ubuntu.com/
```

Comment:
Sadly, no. But the packages are verified against the Ubuntu keyring before extracting